source/content/guides/upgrade-drupal-8-ic-to-drupal-9.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Upgrade Drupal 8 with Integrated Composer to Drupal 9
-description: Upgrade Pantheon Drupal 8 Sites with Integrated Composer to Drupal 9.
+title: Upgrade Drupal 8 with Integrated Composer to the Latest Version of Drupal
```

Comment:
@stovak , should it be "Learn how to update a site to the latest version of Drupal" (omitting ref to D8)?

Reply:
yes. All of these yes.